python/ycm/client/command_request.py

```diff
@@ -69,9 +71,50 @@ def RunPostCommandActionsIfNeeded( self ):
         vimsupport.JumpToLocation( self._response[ 'filepath' ],
                                     self._response[ 'line_num' ],
                                     self._response[ 'column_num' ] )
-    elif 'message' in self._response:
-      vimsupport.EchoText( self._response['message'] )
+    elif self._is_fixit_command:
```

Comment:
have put each section in its own method. Thought about doing something like removing `is_goto_command` and `is_fixit_command` and just storing a callable as the `handler`, but that seemed overkill (at least for now)